client/components/Splash.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```

Comment:
why are you removing PropTypes?

Reply:
It wasn't being used. Jessa said to, it was giving us and error in the console bcus it wasn't being used